aws-cli/bash-linux/dynamodb/tests/test_dynamodb_examples.sh

```diff
@@ -0,0 +1,361 @@
+#    SPDX-License-Identifier: Apache-2.0
+#
+###############################################################################
+
+###############################################################################
+#
+#     Before running this AWS CLI example, set up your development environment, including your credentials.
+#
+#     For more information, see the following documentation topic:
+#
+#     https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html
+#
+###############################################################################
+
+# Set default values.
+
+###############################################################################
+# function main
+#
+# This function runs the Amazon DynamoDB examples' tests.
+###############################################################################
+function main() {
+  # bashsupport disable=BP2001
+  export INTERACTIVE=false
+  # bashsupport disable=BP2001
+  export VERBOSE=false
+
+  source ./include_tests.sh
+  {
+    local current_directory
+    current_directory=$(pwd)
+    cd ..
+    source ./dynamodb_operations.sh
+    source ./scenario_getting_started_movies.sh
+    # shellcheck disable=SC2164
+    cd "$current_directory"
+  }
+
+  ###############################################################################
+  # function usage
+  #
+  # This function prints usage information for the script.
+  ###############################################################################
+  function usage() {
+    echo "This script tests Amazon DynamoDB operations in the AWS CLI."
+    echo ""
+    echo "To pause the script between steps, so you can see the results in the"
+    echo "AWS Management Console, include the parameter -i."
+    echo ""
+    echo "IMPORTANT: Running this script creates resources in your Amazon"
+    echo "   account that can incur charges. It is your responsibility to"
+    echo "   ensure that no resources are left in your account after the script"
+    echo "   completes. If an error occurs during the operation of the script,"
+    echo "   then resources can remain that you might need to delete manually."
+  }
+
+  local option OPTARG # Required to use getopts command in a function.
+
+  # Retrieve the calling parameters
+  while getopts "ivh" option; do
+    case "${option}" in
+      i)
+        INTERACTIVE=true
+        VERBOSE=true
+        iecho
+        ;;
+      v) VERBOSE=true ;;
+      h)
+        usage
+        return 0
+        ;;
+      \?)
+        echo "Invalid parameter"
+        usage
+        return 1
+        ;;
+    esac
+  done
+
+  if [ "$INTERACTIVE" == "true" ]; then iecho "Tests running in interactive mode."; fi
+  if [ "$VERBOSE" == "true" ]; then iecho "Tests running in verbose mode."; fi
+
+  iecho "***************SETUP STEPS******************"
+  local test_table_name
+  test_table_name=$(generate_random_name testcli)
+  local test_provisioned_throughput="ReadCapacityUnits=5,WriteCapacityUnits=5"
```

Comment:
Same question about these filenames, are there sample files the user would need to run the tests?

Reply:
Same as above. These are temp files. Maybe next time I will use a temp folder.